contracts/events/EventFactory.sol

```diff
@@ -14,7 +16,8 @@ contract EventFactory {
         uint16 indexed _version,
         address indexed _topicAddress, 
         bytes32[10] _name, 
-        bytes32[10] _resultNames);
+        bytes32[11] _resultNames,
+        uint8 _numOfResults);
```

Comment:
do we really need this? as discussed, we can stop at where it is empty

Reply:
this is for the oracles. certain methods need to check if it is a valid result (ie. result indexes 0-2 are valid). and I am not passing the result names to the oracles as its not used there so we need the num of results.